src/ui/public/vislib/lib/data.js

```diff
@@ -134,8 +136,10 @@ export function VislibLibDataProvider(Private) {
         const id = axis.axisConfig.get('id');
         stackedData[id] = this.getStackedSeries(chartConfig, axis, data, i === 0);
         stackedData[id] = this.injectZeros(stackedData[id], handler.visConfig.get('orderBucketsBySum', false));
+        stackedData[id] = markZeros(stackedData[id]);
         axis.axisConfig.set('stackedSeries', stackedData[id].length);
         axis.stack(_.map(stackedData[id], 'values'));
```

Comment:
could we rather fix the stacking implementation to take this into account ? so if total value is 0, we should just leave it at 0, and not be smart and say that each of the series is now taking an equal portion of that 0